play/PlayerRules/CharacterCreation/03_Feats.txt

```diff
@@ -527,6 +527,73 @@ Prerequisites: Level 10, Twitch
 You may aim at an enemy as a free action instead of one action. Only 
 works with CQB weapons. No Miss Chance penalty.
 
+====================
+Primary Stat Bonus Feats
+====================
+These feats cannot be bought. They are granted for free if you fulfill their prerequisites.
+
+Lucky: 
+Prerequisites: Luck 6
+	Good things predictably happen to you.
+	Luck = 6	
+		You can choose to reroll a die, once per long rest. You can only reroll 
+		once per separate die use and if you reroll, you must keep the second roll.

```

Comment:
I don't know... I thought Taylor made reference to it. I'll add it to the basic rules